### PR TITLE
Chef install morden and varia themes for supporting.o.o

### PR DESCRIPTION
**Pull request by grischard**

### CODE REVIEW

File: cookbooks/civicrm/recipes/default.rb [hunk 1]
```diff
@@ -91,6 +91,18 @@
   site "supporting.openstreetmap.org"
 end
 
+wordpress_theme "morden" do
+  theme "morden"
```
Comment (tomhughes) [inline cookbooks/civicrm/recipes/default.rb:95]: You don't need to repeat the theme name if it's the same as the resource name - it's the name property so is defaulted automatically.

**Review by tomhughes — CHANGES_REQUESTED** [review]

**Comment by Firefishy** [comment]
I am wondering if switching the code to [wp-cli theme install](https://developer.wordpress.org/cli/commands/theme/install/)  might be a better option.

**Comment by tomhughes** [comment]
> I am wondering if switching the code to [wp-cli theme install](https://developer.wordpress.org/cli/commands/theme/install/) might be a better option.

We already discussed that - does that actually do the install itself or does it just call the API on the site and ask it to do the install, which will then have the same permissions issues as using the web interface?

**Comment by grischard** [comment]
@Firefishy this also works, and the wp-cli path runs into handling output, permissions, etc. Unless we can think of concrete advantages to doing this with wp-cli, let's not let perfect be the enemy of good?

**Pull request merged**